src/sql/workbench/contrib/query/browser/queryEditor.ts

```diff
@@ -257,25 +257,33 @@ export class QueryEditor extends BaseEditor {
 	private setTaskbarContent(): void {
 		// Create HTML Elements for the taskbar
 		let separator = Taskbar.createTaskbarSeparator();
-
-		// Set the content in the order we desire
-		let content: ITaskbarContent[] = [
-			{ action: this._runQueryAction },
-			{ action: this._cancelQueryAction },
-			{ element: separator },
-			{ action: this._toggleConnectDatabaseAction },
-			{ action: this._changeConnectionAction },
-			{ action: this._listDatabasesAction },
-			{ element: separator },
-			{ action: this._estimatedQueryPlanAction },
-			{ action: this._toggleSqlcmdMode },
-			{ action: this._exportAsNotebookAction }
-		];
-
+		let content: ITaskbarContent[];
 		// Remove the estimated query plan action if preview features are not enabled
 		let previewFeaturesEnabled = this.configurationService.getValue('workbench')['enablePreviewFeatures'];
-		if (!previewFeaturesEnabled) {
-			content.splice(7, 1);
+		if (previewFeaturesEnabled) {
+			content = [
+				{ action: this._runQueryAction },
+				{ action: this._cancelQueryAction },
+				{ element: separator },
+				{ action: this._toggleConnectDatabaseAction },
+				{ action: this._changeConnectionAction },
+				{ action: this._listDatabasesAction },
+				{ element: separator },
+				{ action: this._estimatedQueryPlanAction }, // Preview
+				{ action: this._toggleSqlcmdMode },
```

Comment:
I believe SQLCMD is also preview in release/1.21.  Is it intentional to make this command no longer preview?

Reply:
Nope, good catch. The comment wasn't updated when sqlcmd was added and so I missed excluding that when I modified the code originally.